### ISSUE TITLE
02_a2r_demo

### ISSUE THREAD

**Issue by troettge** [learnB4SS/slides#14]
@jvcasillas:

S2: “As we just heard from Timo”, we don’t actually, right? Bayes theorem comes after

So the narrative here should be something like: Before we actually tackle the conceptual background, we will get our hand dirty immediately to lose our fear. 

S4: Worth explaining the nature of standard error (so you can relate it to Bayesian equivalent?) Maybe too much..

I think there might be a lot of question marks here: What do plausible lines mean? where do they come from? Why are they different? what is a posterior?

I like the walking through the dataset part. Maybe we should do that everytime we introduce new variables / measures.

S9: This might confuse people. When they hear interecept they think random intercept. Maybe worth talking about estimating the mean of the measurement without any predictors instead. What is a parameter?

S10: explain piping %>%

S13: Code chunk on the right goes out of bounce

Overall, people might be confused to talk about the estimation sigma at this point. Maybe we can drop that? If not explain what sigma conceptually represents (the variance)?

**Comment by jvcasillas** [comment]
> S2: “As we just heard from Timo”, we don’t actually, right? Bayes theorem comes after
> 

Yes. This was a mistake (I hadn't watched your updated videos yet). 

> So the narrative here should be something like: Before we actually tackle the conceptual background, we will get our hand dirty immediately to lose our fear.
> 

Precisely. 


> S4: Worth explaining the nature of standard error (so you can relate it to Bayesian equivalent?) Maybe too much..
> 

Not a problem. 

> I think there might be a lot of question marks here: What do plausible lines mean? where do they come from? Why are they different? what is a posterior?
> 

Yes, there should be question marks here. In essence, the idea is to start building intuition about what it means to have more/less plausible estimates. I think "where do they come from?", "why are they different?", and "what is a posterior?" are exactly the questions I want them to be asking themselves at this point, but I think you make a good point that perhaps a bit more setup is needed. I have added 3 slides before this. Take a look and see if this helps. In any case, my hope is that they will build some of that intuition in the upcoming slides. They are going to have to get their hands dirty with the terminology anyway, so may as well jump in. I've been (will be) careful to emphasize what the key ideas to hold on to are over the course of the slides. 


> I like the walking through the dataset part. Maybe we should do that everytime we introduce new variables / measures.
> 
> S9: This might confuse people. When they hear interecept they think random intercept. Maybe worth talking about estimating the mean of the measurement without any predictors instead. What is a parameter?
> 
> S10: explain piping %>%
> 
> S13: Code chunk on the right goes out of bounce
> 
> you also refer to ar_bayes_int in the code chunk, which was not defined earlier on the slides. More descriptive names would help to avoid confusion maybe
> 

Note that most of these slides I won't actually present (since it will be a live coding demo). They are there for completeness, you could say... so that you and stef know what I plan to talk about, and for those that might not see the demo... or for those that want something to refer to before, after or doing the workshop. I plan on actually showing the slides for 1-9 (1-5 in the first version you saw) and 16-end. None of those slides are there so that they learn to do what is in the rcode (practical), but rather for the conceptual info (theoretical). Perhaps I should just make it echo=F for all of those. 

> Overall, people might be confused to talk about the estimation sigma at this point. Maybe we can drop that? If not explain what sigma conceptually represents (the variance)?

You can't fit an intercept only model without estimating sigma in a bayesian framework (I don't think so, anyway). Or do you mean to take it out of the printed data frame? In any case I plan on making it very clear that it is just an estimate for SD of articulation_rate (which we calculate by hand in the slide before and will do in the live demo).

**Comment by troettge** [comment]
Totally misinterpreted the slides as slides per se, but if they stand in for your live coding, most of my comments are redundant. Sounds all really good. Keen to see it in live action :)
